Native Compiles but does not work missing  com.sun.xml.internal.ws.spi.ProviderImpl

I am testing the CXF Quarkus extension with:

- Quarkus 1.13.2
- CXF 3.4.3
- CXF Quarkus 0.5
- JDK11

Running in dev mode or the jar build works fine but after building a native image it fails with:

`RESTEASY002020: Unhandled asynchronous exception, sending back 500: org.jboss.resteasy.spi.ApplicationException: javax.xml.ws.WebServiceException: Provider com.sun.xml.internal.ws.spi.ProviderImpl not found`

```
Caused by: java.lang.ClassNotFoundException: com.sun.xml.internal.ws.spi.ProviderImpl
        at com.oracle.svm.core.hub.ClassForNameSupport.forName(ClassForNameSupport.java:60)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:281)
        at javax.xml.ws.spi.ServiceLoaderUtil.nullSafeLoadClass(ServiceLoaderUtil.java:60)
        at javax.xml.ws.spi.ServiceLoaderUtil.safeLoadClass(ServiceLoaderUtil.java:93)
        at javax.xml.ws.spi.ServiceLoaderUtil.newInstance(ServiceLoaderUtil.java:71)
        ... 66 more
```

I am guessing this has something to do with me missing a dependency or the fact I am building on JDK11 and that is not available.

I am pretty much using the POM's in this project as the basis.

The only significant changes in my POM are the inclusions of RESTEasy with Reactive (I call the service inside a reactive method and this works in dev or jar builds):

```
        <dependency>
            <groupId>io.quarkus</groupId>
            <artifactId>quarkus-resteasy-jackson</artifactId>
        </dependency>
        <dependency>
            <groupId>io.quarkus</groupId>
            <artifactId>quarkus-resteasy-mutiny</artifactId>
        </dependency>
```

Any suggestions?

--- a/deployment/src/main/java/io/quarkiverse/cxf/deployment/QuarkusCxfProcessor.java
+++ b/deployment/src/main/java/io/quarkiverse/cxf/deployment/QuarkusCxfProcessor.java
@@ -800,6 +800,8 @@ class QuarkusCxfProcessor {
                 "com.sun.xml.internal.bind.retainReferenceToInfo"));
         reflectiveItems.produce(new ReflectiveClassBuildItem(false, false,
                 //manually added
+                "org.apache.cxf.jaxws.spi.ProviderImpl",
+                "com.sun.xml.bind.v2.JAXBContextFactory",
                 "org.apache.cxf.wsdl.interceptors.BareInInterceptor",
                 "com.sun.msv.reader.GrammarReaderController",
                 "org.apache.cxf.binding.soap.interceptor.RPCInInterceptor",
@@ -983,6 +985,7 @@ class QuarkusCxfProcessor {
         //TODO add @HandlerChain (file) and parse it to add class loading
         return new NativeImageResourceBuildItem("com/sun/xml/fastinfoset/resources/ResourceBundle.properties",
                 "META-INF/cxf/bus-extensions.txt",
+                "META-INF/services/javax.xml.ws.spi.Provider",
                 "META-INF/cxf/cxf.xml",
                 "META-INF/cxf/org.apache.cxf.bus.factory",
                 "META-INF/services/org.apache.cxf.bus.factory",

